packages/react/src/components/table/table-row.tsx

```diff
@@ -103,7 +103,7 @@ const StyledTableRow = styled.tr<StyledTableRowProps>`
         position: relative;
 
         &::after {
-            box-shadow: inset 0 0 0 1px ${theme.notifications['alert-2.1']};
+            box-shadow: inset 0 0 0 1px ${theme.component['table-error-row-border-color']};
```

Comment:
Je pense que j'inverserais, le error est spécifique sur la row
```suggestion
            box-shadow: inset 0 0 0 1px ${theme.component['table-row-error-border-color']};
```